usaspending_api/awards/management/commands/loadcontracts.py

```diff
@@ -18,6 +18,12 @@ def add_arguments(self, parser):
         parser.add_argument('file', nargs=1, help='the file to load')
 
     def handle(self, *args, **options):
+        # Create a new submission attributes object for this timestamp
```

Comment:
I've put a few reminders throughout re: docstrings instead of comments to document a Python function. Small thing that doesn't need to hold up a merge, just noting.